addons/sale/models/onboarding_step.py

```diff
@@ -0,0 +1,120 @@
+# coding: utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+
+from odoo import _, api, Command, models
+from odoo.modules.module import get_module_resource
+
+
+class OnboardingStep(models.Model):
+    _inherit = 'onboarding.onboarding.step'
+
+    @api.model
+    def action_save_sale_quotation_onboarding_layout_step(self):
+        """:return: Whether this step was validated for the first time."""
+        step = self.env.ref('sale.sale_quotation_onboarding_layout_step',
+                            raise_if_not_found=False)
+        return bool(step.action_set_just_done()) if step else False
+
+    @api.model
+    def action_open_sale_onboarding_payment_provider(self):
+        self.env.company.get_chart_of_accounts_or_fail()
+        return self.env["ir.actions.actions"]._for_xml_id("sale.action_open_sale_payment_provider_onboarding_wizard")
```

Comment:
```suggestion
        return self.env['ir.actions.actions']._for_xml_id('sale.action_open_sale_payment_provider_onboarding_wizard')
```
We keep double quoted strings for strings shown to the user/docstrings, not for pure tech strings ;).